ast/ddl.go

```diff
@@ -363,7 +364,19 @@ type IndexOption struct {
 
 // Restore implements Node interface.
 func (n *IndexOption) Restore(ctx *RestoreCtx) error {
-	return errors.New("Not implemented")
+	if n.KeyBlockSize > 0 {
+		ctx.WriteKeyWord(" KEY_BLOCK_SIZE = ")
+		ctx.WritePlain(strconv.FormatUint(n.KeyBlockSize, 10))
+	}
+	if n.Tp == model.IndexTypeBtree || n.Tp == model.IndexTypeHash {
```

Comment:
```suggestion
	if n.Tp != model.IndexTypeInvalid {
```

or change to a `switch` statement, so it won't silently fail even if there are suddenly new index types.